internals/notifier.py

```diff
@@ -146,16 +146,12 @@ def apply_subscription_rules(
   return results
 
 
-def make_email_tasks(fe: FeatureEntry, is_update: bool=False,
-    changes: Optional[list]=None):
+def make_email_tasks(fe: FeatureEntry, target_emails: list[str],
```

Comment:
I think you need to keep this function as-is for the existing email change notifications, and then create a new, separate function to make email tasks for the reviewers.  The reason is that for the reviews, we don't want to notify blink component owners, feature starrers, or anyone else other than the reviewers.

Reply:
Done